README.md

```diff
@@ -67,7 +67,8 @@ NOTE: To update the component status:
 | Components             | Design | Issue | Web Components     | Angular Integration | Blazor Integration |
```

Comment:
The original push for making underline hidden by default was to align with the breadcrumb. Now that the anchor is decoupled from the breadcrumb that really feels like the wrong default. i.e. just adding a link to a page is indistinguishable by default: 

![image](https://user-images.githubusercontent.com/1588923/207425283-b0262a27-4198-4ece-bf7d-5d8546f88a2d.png)

I'll ask in the team meeting but I think we should switch to the `underline-hidden` attribute and show the underline by default.

Reply:
Talked about it in the team meeting and nate agreed that the default style looking like "standalone quiet", i.e. indistinguishable from normal test is not the right default.
We should show the underline by default and change the attribute to `underline-hidden`. We also in the future need to revisit the "standalone quiet" state because it needs some affordance to indicate that it is a link.

Reply:
Changed to `underline-hidden`